client/src/locales/en-US.json

```
@@ -774,12 +774,12 @@
                 "summary": "User has been added successfully!"
             },
             "subtitles": {
-                "waitForGuest1": "To onboard a new user, you have to be in direct contact with them throughout the onboarding process.",
-                "waitForGuest2": "Please don't close this window during the process",
+                "waitForGuest1": "To onboard a new user, you have to be in direct contact with them throughout the onboarding process (around 2 minutes).",
```

Kudos for the gender-neutral "them" here! (idk if it was wanted ^^)
```suggestion
                "waitForGuest1": "To onboard a new user, you have to be in direct contact with them during the onboarding process (around 2 minutes).",
```